auth_admin_passkey/model/res_config.py

```diff
@@ -1,76 +1,53 @@
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013-2014 GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from openerp.osv import fields
-from openerp.osv.orm import TransientModel
-from openerp.tools.safe_eval import safe_eval
+from openerp import api, fields, models
 
 
-class base_config_settings(TransientModel):
+class BaseConfigSettings(models.TransientModel):
     _inherit = 'base.config.settings'
 
     # Getter / Setter Section
-    def get_default_auth_admin_passkey_send_to_admin(
-            self, cr, uid, ids, context=None):
-        icp = self.pool['ir.config_parameter']
+    @api.model
+    def get_default_auth_admin_passkey_send_to_admin(self, fields):
         return {
-            'auth_admin_passkey_send_to_admin': safe_eval(icp.get_param(
-                cr, uid, 'auth_admin_passkey.send_to_admin', 'True')),
+            'auth_admin_passkey_send_to_admin':
+            self.env["ir.config_parameter"].get_param(
+                "auth_admin_passkey.send_to_admin")
         }
 
-    def set_auth_admin_passkey_send_to_admin(self, cr, uid, ids, context=None):
-        config = self.browse(cr, uid, ids[0], context=context)
-        icp = self.pool['ir.config_parameter']
-        icp.set_param(
-            cr, uid, 'auth_admin_passkey.send_to_admin',
-            repr(config.auth_admin_passkey_send_to_admin))
+    @api.multi
+    def set_auth_admin_passkey_send_to_admin(self):
+        for config in self:
+            self.env['ir.config_parameter'].set_param(
+                "auth_admin_passkey.send_to_admin",
+                config.auth_admin_passkey_send_to_admin or '')
 
-    def get_default_auth_admin_passkey_send_to_user(
-            self, cr, uid, ids, context=None):
-        icp = self.pool['ir.config_parameter']
+    @api.model
+    def get_default_auth_admin_passkey_send_to_user(self, fields):
         return {
-            'auth_admin_passkey_send_to_user': safe_eval(icp.get_param(
-                cr, uid, 'auth_admin_passkey.send_to_user', 'True')),
+            'auth_admin_passkey_send_to_user':
+            self.env["ir.config_parameter"].get_param(
```

Comment:
Hi,
Why did you removed safe_eval here, and not here : 
https://github.com/OCA/server-tools/pull/519/commits/8025a5db364bb551160a32e5f763081e70111b68#diff-70199c6feb20b249a2794f26b51b5105R35